CheatSheets/kotlin-cheatsheet.md

```diff
@@ -0,0 +1,172 @@
+# PHP Cheatsheet for Developers
+
+## Table of Contents
```

Comment:
The hyperlinks for Basic, If Else, While loop, and Collections aren't working properly!

Reply:
ok, I'll rectify it!